app/views/layouts/application.html.erb

```diff
@@ -21,6 +21,10 @@
       <div class="navbar navbar-dark bg-primary">
         <div class="container ">
           <%=link_to 'ISTOS.CY', root_path, class: "navbar-brand" %>
+          <% if current_user&.support_user?  %>
+            <%= link_to "#{t('support_portal')}", support_interface_root_path, class: 'btn btn-outline-secondary' %>
+          <% end %>
+
```

Comment:
Can we remove this completely. Let's maintain a clear separation of the visitor facing site from support.